2-0-data-structures-and-algorithms/src/test/java/com/bobocode/array_list/ArrayListTest.java

```diff
@@ -37,20 +40,23 @@ public void sizeOfEmptyArrayWrapper() {
     @Test
     @Order(3)
     void size() {
-        arrayList.add(10);
-        arrayList.add(15);
-        arrayList.add(20);
-
+        try {
```

Comment:
@erneman1 **test should not contain these** `try-catch` blocks. Please create a helper method `setTestSize(size)` and move this logic to that method. You can also mark that method with [@SneakyThrows](https://projectlombok.org/features/SneakyThrows)